house_of_refuge/frontend/src/scripts/index.js

```diff
@@ -143,7 +144,21 @@ const App = ({subs, userData, coordinators, helped}) => {
 
   console.log("query:", searchParams.toString(), peopleFilter);
 
+  useEffect(() => {
+    const handler = () => {
+      setTabVisible(!document.hidden);
+    };
+    document.addEventListener('visibilitychange', handler);
+    return function cleanupListener() {
+      document.removeEventListener('visibilitychange', handler);
+    };
+  });
```

Comment:
we should pass an empty Array here, right? No need to rerun this on each rernder, right? Could be run once?